Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -172,8 +172,27 @@ class ClearPrivateDataTableViewController: UITableViewController {
              }
              }
              } else {*/
+            
+            // Reset Webkit configuration to remove data from memory
+            if clear.contains(where: { $0 is CookiesClearable || $0 is CacheClearable }) {
+                self.tabManager.resetConfiguration()
+                // Unlock the folders to allow clearing of data.
+                if Preferences.Privacy.blockAllCookies.value {
+                    _ = FileManager.default.setFolderAccess([
```

Comment:
Could we add `@discardableResults` to the `setFolderAccess` method?

Reply:
updated